README.md

```diff
@@ -120,41 +124,193 @@ Each flavor represents a pair of programming language and builds tool. All the u
+Create-cycle-app comes pre-packaged with this core flavor:
 
 * [cycle-scripts](https://github.com/cyclejs-community/create-cycle-app/blob/master/packages/cycle-scripts)
 
-### Custom flavour
+### Using a custom flavor
 
 Custom flavors allow generating starting projects to fulfil specific needs.
-They can be published to npm, or being used locally via the create-cycle-app CLI.
+They can be published to npm or a git repo or even be referenced locally (esp. useful when you create your _own_ custom flavor - see below)
 
 #### How to use custom flavors
 
 ##### Providing your own
-When creating a project, you can inform which flavor you want to use with the `--flavor` flag:
+When creating a project, you can specify which flavor to use with the `--flavor` flag:
 
 ```sh
 $ create-cycle-app <name> --flavor <flavor>
 ```
-Some examples of how a flavor could be specified:
 
+The flavor will be installed via `npm install` and can therefore reference any flavor resource installable via `npm`
+
+Some examples of how a flavor can be specified:
+
+*npm name*
+
+```sh
+$ create-cycle-app my-app --flavor cycle-scripts-one-fits-all
 ```
-$ create-cycle-app my-app --flavor cycle-scripts-es-webpack
 
-$ create-cycle-app my-app --flavor cycle-scripts-es-webpack@x.y.z
+*npm name with specific version*
 
-$ create-cycle-app my-app --flavor ./relative/path/to/cycle-scripts-es-webpack
+```sh
+$ create-cycle-app my-app --flavor cycle-scripts-one-fits-all@1.1.0
+```
+
+*github account/repo*
+
+```sh
+$ create-cycle-app my-app --flavor github:my-account/my-cycle-flavor
+```
+
+*reference to local file path*
+
+```sh
+$ create-cycle-app my-app --flavor ./relative/path/to/my-cycle-flavor
 ```
 
 ##### Flavors:
 
 - Check [available flavors](https://github.com/cyclejs-community/create-cycle-app-flavors)
 
+### Creating a custom flavor
+
+Copy an existing flavor to a local folder and reference via the local file path.
+
+Example: Extend default `cycle-scripts` flavor
+
+When you create a new project, the default `cycle-scripts` is installed in `node_modules` of your project. You can simply copy this package and work from there, using it as a simple base.
+
+```sh
+$ cp node_modules/cycle-scripts ./cycle-flavors
+```
+
+The default flavor package has the following structure:
+
+```
+/scripts
+  /configs
+    .babelrc
+  build.js
+  eject.js
+  init.js
+  start.js
+  test.js
+
+/template
+  /public
+  /src
+index.js
+package.json
+Readme.md
+```
+
+## Single flavor from lerna project
+
+If you want to use a [lerna](https://github.com/lerna/lerna) repo with multiple packages, you can make a [sparse checkout](https://gist.github.com/sumardi/5559896) of the specific package you want to use as the base.
+
+We recommend you use [one-fits-all](https://github.com/cyclejs-community/create-cycle-app-flavors/tree/master/packages/cycle-scripts-one-fits-all) as a baseline (note: it uses [webpack-blocks](https://github.com/andywer/webpack-blocks))
```

Comment:
I would try to keep as a basic suggestion the core flavor that come with cca, we could definitely point out to `one-fits-all` as an example for a more articulated flavor perhapsh? This simply  because by doing so will make the readme a bit more future-proof, let's say one-fit-all will become core, or the core [flavor will evolve](https://github.com/cyclejs-community/create-cycle-app/issues/85)...